### PR TITLE
DOCSP-38873 2.2.5 mongosh rn

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
2.2.5 mongosh rn

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/mongodb-shell/DOCSP-38873/changelog/#v2.2.5

## JIRA
https://jira.mongodb.org/browse/DOCSP-38873?filter=-1

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6629512db67dbf804aed70bf

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,23 @@ Release Notes
...
    :class: singlecol
 
+v2.2.5
+------
+
+*Released April 22, 2024*
+
+Performance improvements:
+
+- :issue:`MONGOSH-1759` – Updates the stance on the trade-off between
```
Comment (jason-price-mongodb) [inline source/changelog.txt:22]: Consider rewrite to not use "stance" and "trade-off"

File: source/changelog.txt [hunk 2]
```diff
@@ -12,6 +12,23 @@ Release Notes
...
 
+v2.2.5
+------
+
+*Released April 22, 2024*
+
+Performance improvements:
+
+- :issue:`MONGOSH-1759` – Updates the stance on the trade-off between 
+  startup performance and binary size. ``mongosh`` is now a couple dozen
```
Comment (jason-price-mongodb) [inline source/changelog.txt:23]: Ditto on couple of dozen :)

**Review by jason-price-mongodb — APPROVED** [review]
@jocelyn-mendez1 somewhat lgtm but consider rewrites of notes. Cheers.

File: source/changelog.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/changelog.txt]: Updated.

File: source/changelog.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/changelog.txt]: updated.

File: source/changelog.txt [hunk 3]
```diff
@@ -12,6 +12,23 @@ Release Notes
...
    :class: singlecol
 
+v2.2.5
+------
+
+*Released April 22, 2024*
+
+Performance improvements:
+
+- :issue:`MONGOSH-1759` – Updates the policy regarding the balance between
```
Comment (GaurabAryal) [inline source/changelog.txt:22]: Could we reword this to:
Improves mongosh startup time time by ~40% by disabling startup snapshot compression in boxednode.

**Review by GaurabAryal — CHANGES_REQUESTED** [review]
Small rewording but otherwise LGTM

File: source/changelog.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/changelog.txt:22]: Updated.

Comment (jocelyn-mendez1) [reply source/changelog.txt:22]: Sorry @GaurabAryal, I just wanted to confirm we wanted the second sentence updated?

**Comment by mdb-ashley** [comment]
Per @jocelyn-mendez1 , @GaurabAryal LGTMed via slack.
